### ISSUE TITLE
Linux support?

### ISSUE THREAD

**Issue by HunterCZ122** [ZooLSmith/MIGI3#1]
I think it's too early, but Linux support would be more appreciated.
Python is cross-platform, so maybe it shouldn't be problem.

**Comment by ZooLSmith** [comment]
Linux support is planned yes, it's indeed one of the reasons I've chosen to go with python+angular. I'll still have to check how vpk_linux32 works as the python vpk lib I'm using is broken for writing files and doesn't support saving as multichunk vpks

**Comment by ZooLSmith** [comment]
Yeah I'm getting issues with pywebview, it just doesn't wanna run the Angular stuff, it's as if you were opening it directly from the desktop without going through a server etc.. it just gives a white page.
I've been looking for a solution for hour and it doesn't seem like there's one.
I might just make a console version for the Linux version.. not even counting the amount of issues there is with the fact that Valve's tools are not really ported over to Linux.. it's a mess.
Most of my functionalities do work.. but yeah looks like you guys will have a CLI version instead of standard GUI.
